src/screens/HomeScreen.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import { StyleSheet, Text, TouchableOpacity, View } from 'react-native'
+import { Icon } from 'react-native-elements'
+
+const HomeScreen = props => {
+  return (
+
+    <View style={styles.container}>
+      <View style={styles.body}>
+
```

Comment:
Please remove these empty lines here but keep empty lines between `<View></View>` blocks.
It seems our linter does not pick this up. I will check for some JSX lint but for now lets make this code a bit more structured manually.